docs/GCP_CLOUDRUN.md

```diff
@@ -51,6 +52,68 @@ Similarly, if there are existing role bindings, which you would like to add a me
 
 More information can be found in the terraform [documentation](https://www.terraform.io/docs/providers/google/r/cloud_run_service_iam.html).
 
+## Secrets
+Secrets can be attached to the cloudrun service either as environment variables or mounted as volumes.
```

Comment:
let's recommend against them being used in environment variables only as a last resort due to inherent security risks.

Reply:
Have updated